pkg/kinflate/resource/secret_test.go

```diff
@@ -26,21 +26,19 @@ import (
 	manifest "k8s.io/kubectl/pkg/apis/manifest/v1alpha1"
 )
 
-func TestNewFromSecretGenerator(t *testing.T) {
-	secret := manifest.SecretGenerator{
+func makeSecretGenerator() manifest.SecretGenerator {
```

Comment:
Please no, the goal was to keep the data where it's tested (remove logic from tests).

Reply:
What logic do you want to remove here?

Reply:
Discussed offline.
Will update it.